### PR TITLE
Feature/kc/1

### LINKED ISSUE #1: Working copy of EXPO APP
branch name: `feature/XXX/1`

* [ ] read expo doc
* [ ] differences in expo vs clean react native
* [ ] configure eslint/prettier

AC:
* [ ] typescript
* [ ] working
* [ ] android
* [ ] ios
* [ ] web
* [ ] simple page
* * [ ] title: "HELLO"
* * [ ] desc: "Lorem ipsum..."
* [ ] update README.md with instruction how to develop app

### PR DESCRIPTION
**Pull request by krzysiek-mc**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,30 @@
...
+# Airbnb Clone
+
+That project is about creating Aribnb clone. Its purpose is to make dev team stronger.
+
+## Installation
+
+How to download the project and start coding?
+
+- Go to preferred directory on your computer
+- Run terminal and type commands you can see below
```
Comment (mm-webx) [inline README.md:10]: too much detail

File: README.md [hunk 2]
```diff
@@ -1 +1,30 @@
...
+  $yarn expo start
+```
+
+Then you will be able to select what platform you want to run the app on.
+
+Click
+
+- `i` for running it on iOS emulator (MAC only)
+- `a` for running it on Android emulator
+- `w` for running it on the web
```
Comment (mm-webx) [inline README.md:28]: too much detail - you copy info from terminal (when you run expo start)

File: src/components/Header.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { Text, TextProps, StyleSheet, View } from "react-native";
```
Comment (mm-webx) [inline src/components/Header.tsx:2]: unused imports?

**Review by mm-webx — APPROVED** [review]

**Comment by krzysiek-mc** [comment]
MM PR was merged

**Pull request closed without merging**